### ISSUE TITLE
Add new metrics

### ISSUE THREAD

**Issue by dpascualhe** [JdeRobot/DetectionMetrics#240]
Good starting point: confusion matrix, precision, recall.

**Comment by Ravish990** [comment]
Hello @dpascualhe  I am Ravish . I want to contribute to the organization . I am very excited . Can you please assign the issue to me to get started ?

**Comment by Afreen-Kazi-1** [comment]
Hello @dpascualhe, I’m Afreen from VJTI, Mumbai. I’m a second year Computer Engineering major with a minor in Robotics. I noticed the issue about adding new metrics to the repository and I’m excited to contribute. Please assign this issue to me.

**Comment by Afreen-Kazi-1** [comment]
Alright @dpascualhe. I checked that out. I had prepared the Precision and Recall metrics as different classes inheriting from Metrics. So now, should I update them as functions under the Confusion matrix class? I think I can also add F1 score. 
I also realized that the test folder has empty files. Is it supposed to contain files for testing the metrics?

**Comment by dpascualhe** [comment]
> So now, should I update them as functions under the Confusion matrix class?

Yes, I think it is the more efficient approach. You can use the new `get_accuracy` function as an example.
https://github.com/JdeRobot/DetectionMetrics/blob/29d6cc164582803451d041ab4d5f7131a26143ae/detectionmetrics/utils/metrics.py#L147

> Should I also add F1 score?

It would be nice to have :smile:

**Comment by Shyam-duba** [comment]
is this issue still open. iam ok to work with this
